docs/how-to/authentication-plugin-examples.md

```diff
@@ -7,39 +7,46 @@ pagination_next: getting-started/authentication
 
 # Handle Authentication and Authorization in your Service
 
-Authentication and authorization are important parts of your Amplication service. Amplication provides flexible options for implementing these security measures.
+Authentication and authorization are crucial components of your Amplication service.
 
-This guide will walk you through the process of updating your authentication and authorization setup in your Amplication-generated service.
+By following this guide and exploring the examples, you'll be well-equipped to implement authentication and authorization in your Amplication-generated service using the authentication plugin that best suits your needs.
 
 ## Adding Authentication to Your Service
 
-If you initially created your service without authentication, you can still add it later. Here are the key steps:
+Before adding any specific authentication plugin, you must first enable the Authentication Entity in your service:
 
-1. Go to your service's Plugins page.
-2. Navigate to the "Authentication" category in the left sidebar.
-3. Choose and add an Auth Provider plugin (e.g., JWT, Auth0, etc.) that suits your needs.
-4. Install the "NestJS Auth Module" plugin if it's not already installed. It's a pre-requisite if you're adding Authentication to a Node.js service.
-5. Configure the Authentication Entity in your service settings.
+1. Go to your service's Entities page.
+2. Locate or create an entity to serve as your Authentication Entity (commonly named "User").
+3. In the entity's settings, enable the "Authentication Entity" option.
```

Comment:
We don't have such an option for an entity, but one needs to go to the Service settings (see below), and define the auth entity-
![image](https://github.com/user-attachments/assets/fe6b0f48-a165-4b8c-b25c-83149a9d19fe)